README.md

```diff
@@ -45,7 +45,7 @@ All machine-learning models implemented in q2-sample-classifier support the opti
 
 ![Alt text](./examples/rfe_plot.jpg?raw=true "Recursive feature elimination plot")
 ## Classification
-In a classification problem, we are interested in predicting the class labels for a number of unlabeled samples. The visualizers show overall accuracy scores (e.g., the percentage of times that the true class label is predicted), and also generates confusion matrices, showing how frequently samples belonging to each class are predicted to be the true class or another class. For example, the following depicts how frequently grape samples are classified to the correct Vineyard (94.1% of the time!):
+In a classification problem, we are interested in predicting the class labels for a number of unlabeled samples. The visualizers show overall accuracy scores (e.g., the percentage of times that the true class label is predicted), baseline error (the error rate for a classifier than always guesses the most common class), and error ratio (the n-fold improvement of the trained classifier over baseline error, or ER = baseline error / (1 - accuracy)). It also generates confusion matrices, showing how frequently samples belonging to each class are predicted to be the true class or another class. For example, the following depicts how frequently grape samples are classified to the correct Vineyard (94.1% of the time!):
```

Comment:
👍 
I was just following along with the ratio in qiime1's `supervised_learning.py`... but I think you're right, the ratio of accuracy rather than the ratio of error makes more sense (and is more consistent).